gemini/grounding/intro-grounding-gemini.ipynb

```diff
@@ -430,6 +412,22 @@
         "print(print_grounding_data(response))"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "> Note: For model before Gemini 2.0, please use the `GoogleSearchRetrieval` tool instead)\n",
+        "\n",
+        "```\n",
+        "response = client.models.generate_content(\n",
+        "    model=MODEL_ID,\n",
+        "    contents=PROMPT,\n",
+        "    config=GenerateContentConfig(tools=[GoogleSearchRetrieval()]),\n",
+        ")\n",
+        "\n",
+        "```"
+      ]
```

Comment:
We're trying to remove all references to previous models across our content, so can you remove this section?